system/Throttle/Throttler.php

```diff
@@ -169,10 +168,9 @@ public function check(string $key, int $capacity, int $seconds, int $cost = 1):
 		$tokens += $rate * $elapsed;
 		$tokens  = $tokens > $capacity ? $capacity : $tokens;
 
-		// If $tokens > 0, then we are safe to perform the action, but
+		// If $tokens >= 1, then we are safe to perform the action, but
 		// we need to decrement the number of available tokens.
-		if ($tokens > 0)
-		{
```

Comment:
This should be reverted to match CodeIgniters coding standard.
{ = should be on it's own line.